schemas/dto/record.json

```diff
@@ -71,6 +71,11 @@
       ],
       "default": "ACTUAL"
     },
+    "leaderRecordStatus": {
```

Comment:
Could you add this field to the [example](https://github.com/folio-org/data-import-raml-storage/blob/fef2478f065ec4d4def8aaadb6e7426078b760dd/examples/mod-source-record-storage/record.sample) as well? Thanks

Reply:
Done. Thanks.